wavefront_sdk/entities/histogram/histogram_impl.py

```diff
@@ -358,11 +342,10 @@ def get_min(self):
                 min_val = min(min(dist.percentile(0) for dist
                                   in minute_bin.per_thread_dist.values()),
                               min_val)
-        return min_val if min_val == min_val else None
+        return None if math.isnan(min_val) else min_val
```

Comment:
Same here

Reply:
+/-inf would do the same job, but make more sense, I'd say